src/app/shared/interfaces/notification.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface Notification {
+  type: string,
```

Comment:
add id, generate id from utils directory(you need to create string.utils.ts in shared)

example function:
export function getUID() {
  return (
    String.fromCharCode(Math.floor(Math.random() * 26) + 97) +
    Math.random().toString(16).slice(2) +
    Date.now().toString(16).slice(4)
  );
}